axelor-account/src/main/java/com/axelor/apps/account/db/repo/InvoiceManagementRepository.java

```diff
@@ -76,4 +78,22 @@ public Invoice save(Invoice invoice) {
       throw new PersistenceException(e.getLocalizedMessage());
     }
   }
+
+  @SuppressWarnings({"unchecked", "rawtypes"})
+  @Override
+  public Map<String, Object> populate(Map<String, Object> json, Map<String, Object> context) {
+    try {
+      if (context.get("_model") != null
+          && context.get("_model").toString().contains("SubrogationRelease")) {
+        long id = (long) context.get("id");
+        SubrogationRelease subrogationRelease =
+            Beans.get(SubrogationReleaseRepository.class).find(id);
+        json.put("$SubrogationStatusSelect", subrogationRelease.getStatusSelect());
+      }
+    } catch (Exception e) {
+      e.printStackTrace();
```

Comment:
please trace this exception. printStackTrace or System.out.println should never be used.